examples/Rust library overview.ipynb

```diff
@@ -0,0 +1,500 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "995cd1ee-bbe3-485f-9f7f-0e080aeb56b8",
+   "metadata": {},
+   "source": [
+    "# Introductory tutorial for the synIR library and its python bindings\n",
+    "Welcome to the synIR (/ˈsɪnɚ/) library!\n",
+    "\n",
+    "This jupyter notebook showcases the basic usage of our quantum compilation library."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "1a866fba",
+   "metadata": {},
+   "source": [
+    "## Library overview\n",
+    "This compilation library is a rust-based library with python bindings. \n",
+    "\n",
+    "synIR stands for SYNthesizable Intermediate Representation. Unlike gate-based intermediate representations (e.g. QIR), the library represents quantum circuits as sequences of exponentiated Paulistrings and a Clifford tableau $A$: \n",
+    "\n",
+    "$$U = A\\Pi (\\Pi_{P \\in \\{I, X, Y, Z\\}^N} e^{-\\alpha P})$$\n",
+    "\n",
+    "Here, the inner product of $e^{-\\alpha P}$ is assumed to be mutually commuting. This assumption helps optimize circuits resulting from a trotterization process in exchange for a potential increase of the Trotter Error."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "936dcffd",
+   "metadata": {},
+   "source": [
+    "### Lexicon\n",
+    "Since there are some conflicting terminology in literature, we will define what we mean by some of these terms here in alphabetical order.\n",
```

Comment:
I would sort in a "top-down" order, explaining higher-level concepts first and then dependent concepts.
E.g.
1. Pauli exponential
2. Pauli polynomial
3. Pauli string
4. Pauli letter